components/tools/OmeroWeb/omeroweb/webclient/show.py

```diff
@@ -860,12 +860,12 @@ def paths_to_well_image(conn, params, well_id=None, image_id=None,
                acquisition.id,
                well.id
         from WellSample wellsample
-        left outer join wellsample.details.owner wsowner
+        join wellsample.details.owner wsowner
         left outer join wellsample.plateAcquisition acquisition
-        left outer join wellsample.details.owner aowner
+        left outer join acquisition.details.owner aowner
```

Comment:
@jburel I think if you remove the ```left outer``` from joining acquisition owner then you will not retrieve any Wells that have no PlateAcquisition (since they will also not have any PlateAcquisition owner).